default.py

```diff
@@ -155,6 +166,25 @@ def PLAY(payload):
 def PLAY_SOURCE(payload):
     return control.play_source(urlparse.unquote(payload))
 
+@route('download*')
+def DOWNLOAD(payload):
+    url,name = payload.split("&name=")
```

Comment:
please cleanup and move to new function inside `control.py` `aria2_download(params....)`